src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -26,16 +26,25 @@ object WdlExpression {
 
   val unaryOperators = Set("LogicalNot", "UnaryPlus", "UnaryNegation")
 
-  def evaluate(ast: AstNode, lookup: ScopedLookupFunction, functions: WdlFunctions): Try[WdlValue] = {
+  def interpolate(str: String, lookup: ScopedLookupFunction): String =
+    "\\$\\{([a-zA-Z0-9_]+)\\}".r.findAllIn(str).foldLeft(str) { (string, tag) =>
+      val variable = tag.substring(2, tag.length - 1)
+      val replacement = lookup(variable)
+      string.replace(tag, replacement.valueString)
+    }
+
```

Comment:
I feel even more strongly than usual that this should be a function and not a closure.